### DISCUSSION TITLE
Iteration in the language tour

### DISCUSSION THREAD

**Discussion by qexat** [gleam-lang/gleam#1862] [Ideas & suggestions]
So yesterday I've started to read the language tour and I couldn't find any information about iterating. I'm aware that there is no literal construct for this, but I've had to search outside the docs to find out about `gleam/iterator`. 
My suggestion is that perhaps it should be mentioned somewhere in the tour? Apologies in advance if this is irrelevant. Thank you otherwise!

**Comment by lpil** [comment]
Hey @qexat ! Thanks for checking our Gleam. I agree we need some documentation that explains that Gleam doesn't have any iteration features other than recursion. It is something that I want to cover in detail in the future tutorial, but it's probably worth having a note of this in the language tour also.
